pkg/alert/alert.go

```diff
@@ -203,7 +203,7 @@ func (q *Queue) Push(alerts []*Alert) {
 	q.pushed.Add(float64(len(alerts)))
 
 	// Attach external labels and drop excluded labels before sending.
-	// TODO(bwplotka): User proper relabelling with https://github.com/thanos-io/thanos/issues/660
+	// TODO(bwplotka): User proper relabelling with https://github.com/thanos-io/thanos/issues/660 .
```

Comment:
This is inconsistent. Above you added `.` without space (:  It's up to you but let's make it consistent.

Reply:
There is even full discussion about this ;p https://ux.stackexchange.com/questions/17331/should-a-full-sentence-html-link-include-the-period-in-the-linked-text

I would say no space?

Reply:
Ok, no space it is :+1: